releases/2.13.7.md

```diff
@@ -0,0 +1,66 @@
+* Fix `ArrayOps` bugs (by avoiding `ArraySeq#array`, which does not guarantee element type) ([#9641](https://github.com/scala/scala/pull/9641) by [@som-snytt](https://github.com/som-snytt))
+* Deprecate `IterableOps.toIterable` ([#9774](https://github.com/scala/scala/pull/9774) by [@lrytz](https://github.com/lrytz))
+
+### Other changes
+
+* Accept supplementary Unicode characters in identifiers ([#9687](https://github.com/scala/scala/pull/9687) by [@som-snytt](https://github.com/som-snytt))
+* Add `-Vimplicits-max-modules` to control how types are printed in `-Vimplicits` messages ([#9767](https://github.com/scala/scala/pull/9767) by [@mrdziuban](https://github.com/mrdziuban))
+* Improve tab completion and code assist in REPL ([#9656](https://github.com/scala/scala/pull/9656) by [@retronym](https://github.com/retronym))
+
+Some small changes that will ship in 2.12.16 are also included in this release.
+
+For the complete 2.13.7 change lists, see [all merged PRs](https://github.com/scala/scala/pulls?q=is%3Amerged%20milestone%3A2.13.7) and [all closed bugs](https://github.com/scala/bug/issues?utf8=%E2%9C%93&q=is%3Aclosed+milestone%3A2.13.7).
+
+## Compatibility
+
+As usual for our minor releases, Scala 2.13.7 is binary-compatible with the whole Scala 2.13 series.
+
+Upgrading from 2.12? Enable `-Xmigration` while upgrading to request migration advice from the compiler.
+
+## Contributors
+
+A big thank you to everyone who's helped improve Scala by reporting bugs, improving our documentation, spreading kindness in discussions around Scala, and submitting and reviewing pull requests! You are all magnificent.
+
+This release was brought to you by 25 contributors, according to `git shortlog -sn --no-merges @ ^v2.13.6 ^2.12.x`. Thank you A. P. Marki, Lukas Rytz, Seth Tisue, Jason Zaugg, Jamie Thompson, NthPortal, Georgi Krastev, Guillaume Martres, Dale Wijnand, Martijn Hoekstra, Alec Theriault, Rafał Sumisławski, Matt Dziuban, Li Haoyi, Doug Roper, Sébastien Doeraene, VladKopanev, danicheg, dengziming, megri, nwk37011, Magnolia.K, 梦境迷离, Mathias, James Judd.
+
+Thanks to [Lightbend](https://www.lightbend.com/scala) for their continued sponsorship of the Scala core team’s efforts. Lightbend offers [commercial support](https://www.lightbend.com/lightbend-platform-subscription) for Scala.
+
+## Scala 2.13 notes
+
+The [release notes for Scala 2.13.0](https://github.com/scala/scala/releases/v2.13.0) have important information applicable to the whole 2.13 series.
+
+## Obtaining Scala
+
+Scala releases are available through a variety of channels, including (but not limited to):
+
+* Bump the `scalaVersion` setting in your sbt-based project
+* Download a distribution from [scala-lang.org](http://scala-lang.org/download/2.13.7.html)
+* Obtain JARs via [Maven Central](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.scala-lang%22%20AND%20v%3A%222.13.7%22)
```

Comment:
```suggestion
* Download a distribution from [scala-lang.org](https://scala-lang.org/download/2.13.7.html)
* Obtain JARs via [Maven Central](https://search.maven.org/search?q=g:org.scala-lang%20v:2.13.7)
```

- https://github.com/scala/scala-dev/pull/797

Reply:
@xuwei-k Query filters (i.e. `g:` for group and `v:` for version) must be separated by an `&` (ampersand), not by a `%20` (space).

Reply:
@xuwei-k Query filters (i.e. **`g:`** for group and **`v:`** for version) must be separated by **`%20AND%20`** and not just **`%20`** (space). Tested with Firefox.

Reply:
Thanks. updated

Reply:
thank you